PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -285,19 +291,52 @@ struct HfTaskFlowCharmHadrons {
     }
   }
 
+  /// Compute the Q vector for the candidate's tracks
+  /// \param cand is the candidate
+  /// \param tracksQx is the X component of the Q vector for the tracks
+  /// \param tracksQy is the Y component of the Q vector for the tracks
+  /// \param channel is the decay channel
+  template <typename T1>
+  void getQvecXic0Tracks(const T1& cand,
+                         std::vector<float>& tracksQx,
+                         std::vector<float>& tracksQy,
+                         float& ampl)
```

Comment:
Why is `ampl` passed by non-const reference?

Reply:
Small types are cheap to copy so if you don't need to modify the variable, you should pass it by value.